tests/analyzer_tech_debt.txt

```diff
@@ -80,6 +80,7 @@
 02721_url_cluster
 02534_s3_cluster_insert_select_schema_inference
 02765_parallel_replicas_final_modifier
+02784_parallel_replicas_automatic_decision_join
```

Comment:
This test needs the analyzer supporting JOINs with pure parallel replicas

Reply:
We do not usually add anything to this list. It was created to disallow introducing incompatibility.

Reply:
Parallel replicas with JOINs is not supported with the analyzer yet, so the tests checking the automatic decision on top of that won't work until the underlying feature is implemented. For now I'd rather add tests (with some of those disabled for the analyzer) that not add them at all.

Reply:
Then you'll need to work on supporting it with the analyzer because it'll be a blocker.

Reply:
I know. My point is that it was already a blocker, it's just more tests for when I implement the feature on the new planner.